libs/cart/guides/drivers/magento.md

```diff
@@ -0,0 +1,28 @@
+# Cart Magento Driver
+
+The In-Memory driver is for rapid development without the need to set up a magento/shopify/etc backend. It will mock out the submission of a cart form and operate like a functional backend.
```

Comment:
The is in-memory specific documentation

Reply:
Sorry all the guides were accidentally committed. Please ignore those. Also, Damien should probably be the one to review this.

As an aside though, what's your opinion about splitting Daffodil libs into state subpackages?

Reply:
I was actually going to ask if we could have an issue that describes why we are making these changes. While it seems fine to me, I don't immediately see a really good reason for it.

Reply:
@griest024 could you do a brief write-up on the why's of this?